src/qibo/quantum_info/metrics.py

```diff
@@ -275,6 +275,54 @@ def fidelity(state, target, validate: bool = False):
     return fid
 
 
+def bures_angle(state, target, validate: bool = False):
+    """Calculates the Bures angle :math:`D_{A}` between a ``state`` :math:`\\rho`
+    and a ``target`` state :math:`\\sigma`. This is given by
+
+    .. math::
+        D_{A}(\\rho, \\, \\sigma) = \\text{arccos}\\left(\\sqrt{F(\\rho, \\, \\sigma)}\\right) \\, ,
+
+    where :math:`F(\\rho, \\sigma)` is the :func:`qibo.quantum_info.fidelity`
+    between `state` and `target`.
+
+    Args:
+        state (ndarray): statevector or density matrix.
+        target (ndarray): statevector or density matrix.
+        validate (bool, optional): if ``True``, checks if one of the
+            input states is pure. Defaults to ``False``.
+
+    Returns:
+        float: Bures angle between ``state`` and ``target``.
+    """
+    angle = np.arccos(np.sqrt(fidelity(state, target, validate=validate)))
+
+    return angle
+
+
+def bures_distance(state, target, validate: bool = False):
+    """Calculates the Bures distance :math:`D_{B}` between a ``state`` :math:`\\rho`
+    and a ``target`` state :math:`\\sigma`. This is given by
+
+    .. math::
+        D_{B}(\\rho, \\, \\sigma) = \\sqrt{2 \\, (1 - \\sqrt{F(\\rho, \\, \\sigma)})}
+
+    where :math:`F(\\rho, \\sigma)` is the :func:`qibo.quantum_info.fidelity`
+    between `state` and `target`.
+
+    Args:
+        state (ndarray): statevector or density matrix.
+        target (ndarray): statevector or density matrix.
+        validate (bool, optional): if ``True``, checks if one of the
+            input states is pure. Defaults to ``False``.
+
+    Returns:
+        float: Bures distance between ``state`` and ``target``.
+    """
+    distance = np.sqrt(2 * (1 - fidelity(state, target, validate=validate)))
```

Comment:
Shouldn't there be a square root of the fidelity ($\sqrt{F(\rho, \sigma)}$)?
```suggestion
    distance = np.sqrt(2 * (1 - np.sqrt(fidelity(state, target, validate=validate))))
```